connectors/sqs/element-templates/aws-sqs-connector.json

```
@@ -100,6 +100,64 @@
         "notEmpty": true
       }
     },
+    {
+      "id": "queue.type",
+      "label": "Queue type",
+      "description": "Specify whether the queue is a <a href=\"https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/standard-queues.html\">standard</a> or <a href=\"https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/FIFO-queues.html\">FIFO</a> queue",
+      "group": "queueProperties",
+      "type": "Dropdown",
+      "optional": false,
+      "value": "standard",
+      "choices": [
+        {
+          "name": "Standard",
+          "value": "standard"
+        },
+        {
+          "name": "FIFO",
+          "value": "fifo"
+        }
+      ],
+      "binding": {
+        "type": "property",
+        "name": "queue.type"
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Message Group Id",
```

We use `ID` instead of `Id`.

Also, [reference](https://www.collinsdictionary.com/dictionary/english/id) that ID is actual meaning for identification.

```suggestion
      "label": "Message Group ID",
```